SumoLogic.Logging.Common/Sender/SumoLogicMessageSender.cs

```diff
@@ -123,14 +153,14 @@ private HttpClient HttpClient
         /// <param name="body">The message body.</param>
         /// <param name="name">The message name.</param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
-        public void Send(string body, string name)
+        public void Send(string body)

```

Comment:
Just realized that this is a public API, so we can't change its signature. I will discuss solution with you